runtime/mainnet/src/lib.rs

```diff
@@ -378,13 +384,39 @@ parameter_types! {
 	/// Relay Chain `TransactionByteFee` / 10
 	pub const TransactionByteFee: Balance = fee::TRANSACTION_BYTE_FEE;
 	pub SudoAddress: AccountId = AccountId::from_ss58check("15NMV2JX1NeMwarQiiZvuJ8ixUcvayFDcu1F9Wz1HNpSc8gP").expect("sudo address is valid SS58");
+	pub MaintenancePot: AccountId = AccountId::from_ss58check("1Y3M8pnn3rJcxQn46SbocHcUHYfs4j8W2zHX7XNK99LGSVe").expect("maintenance address is valid SS58");
```

Comment:
Thought about this a little more. `MaintenanceAddress` will be a little more accurate in this case. Pot infers something that can be spent out of, while this is (right now) a multisig address for R0GUE.

Reply:
https://github.com/r0gue-io/pop-node/pull/428/commits/21a8be429c79ab1c66503af2e366f9dfee6d9089